server/.env.example

```diff
@@ -22,3 +22,7 @@ TYPEORM_ENTITIES_DIR=src/entity
 
 # JWT
 JWT_SECRET=mysecret
+
+# NODE_MAILER
+GMAIL_PASSWORD='9J1YYKqj6VEf3RWL'
```

Comment:
Please, do not push secrets. Leave only var names, send secret values to the team chat after merge.